test/python/test_backends.py

```diff
@@ -49,35 +49,35 @@ def test_local_backends_exist(self):
         self.assertTrue(len(local) > 0)
 
     @requires_qe_access
-    def test_remote_backends_exist(self, QE_TOKEN, QE_URL):
+    def test_remote_backends_exist(self, QE_TOKEN, QE_URL, hub=None, group=None, project=None):
         """Test if there are remote backends.
 
         If all correct some should exists.
         """
-        ibmq_provider = IBMQProvider(QE_TOKEN, QE_URL)
+        ibmq_provider = IBMQProvider(QE_TOKEN, QE_URL, hub=None, group=None, project=None)
```

Comment:
Can you check all modified lines similar to this one and pass `hub`, `group`, and `project` without setting them to `None`, as otherwise it seems it seems to kind  defeat the point of adding them to `requires_qe_access`? ie

```
ibmq_provider = IBMQProvider(QE_TOKEN, QE_URL, hub, group, project)
```

Reply:
I wasn't sure if we require Qconfig to specify hub, group, project if the user is not accessing a system which requires it? I was thinking not.